boggle/Boggle_works.py

```diff
@@ -0,0 +1,152 @@
+
+from random import *
+from math import *
+from Gui import *

```

Comment:
"import *" is generally frowned upon these days. Just import the items you use, or import the module only and say Gui.whatever